packages/frontend/src/theme.tsx

```diff
@@ -157,41 +157,19 @@ const brandPalette: PaletteOptions = {
 }
 
 const brandTypography: TypographyOptions = {
-  // fontFamily: 'Montserrat',
-  fontFamily: 'Verdana, sans-serif',
-  button:{
-    fontFamily: 'Montserrat, Verdana, sans-serif',
+  fontFamily: '"Nunito Sans", sans-serif',
+  button: {
+    fontFamily: '"Nunito Sans", sans-serif',
+    textTransform: 'none'
```

Comment:
I don't have a strong opinion about the all caps, I'm fine with removing the textTransform to see how it looks